### DISCUSSION TITLE
Wait until content is loaded control

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#519] [Q&A]
There are loads that can take a long time for the control to update on the screen, is there any progress control until the view is ready?

Is AlertDialog a better option?

**Comment by FeodorFitsner** [comment]
For example?

**Comment by nascin** [comment]
I don't have an example ready here, but imagine that I request a data in a bank, and the response is a little slow, meanwhile my control does not update, I would like loading control, until the control is ready to update the screen .

I saw something in flutter like StreamBuider

**Comment by FeodorFitsner** [comment]
You might not need that. While you are "preparing" (fetching from an API, processing, etc.) the data to display in Flet you can replace some controls with ProgressRing or put it into a overlay using Stack, or just disable part of the form.

**Comment by nascin** [comment]
Hmm, I thought about some of these options, but overlapping with a stack seems to be interesting.

Thanks!
